types/datum_test.go

```diff
@@ -396,6 +397,26 @@ func newRetTypeWithFlenDecimal(tp byte, flen int, decimal int) *FieldType {
 	}
 }
 
+func (ts *testDatumSuite) TestEstimatedMemUsage(c *C) {
+	b := []byte{'a', 'b', 'c', 'd'}
+	enum := Enum{Name: "a", Value: 1}
+	datumArray := []Datum{
```

Comment:
It's better to add a new `Enum` whose underlying type is `Time` into it.

Reply:
Refer [MySQL manual](https://dev.mysql.com/doc/refman/8.0/en/enum.html): An ENUM is a **string** object with a value chosen from a list of permitted values